pyproject.toml

```diff
@@ -7,42 +7,44 @@ name = "supersonic"
 version = "0.1.0"
 description = "GitHub PR automation for modern applications"
 readme = "README.md"
-requires-python = ">=3.9"
 license = "MIT"
 authors = [
     { name = "Cased", email = "ted@cased.com" }
 ]
+requires-python = ">=3.9"
 dependencies = [
-    "PyGithub>=2.1.1",
     "pydantic>=2.0.0",
-    "gitpython>=3.1.40",
-    "aiohttp>=3.8.5",
+    "pygithub>=2.1.1",
     "click>=8.1.3",
+    "gitpython>=3.1.40",
     "jinja2>=3.1.2",
+    "requests>=2.31.0",
+    "types-requests>=2.31.0",
```

Comment:
You could move `types-*` to the dev group.

Reply:
Agreed.